pkg/controller/endpointslice/endpointslice_controller.go

```diff
@@ -69,7 +69,7 @@ const (
 	// defaultSyncBackOff is the default backoff period for syncService calls.
 	defaultSyncBackOff = 1 * time.Second
 	// maxSyncBackOff is the max backoff period for syncService calls.
-	maxSyncBackOff = 100 * time.Second
+	maxSyncBackOff = 1000 * time.Second
```

Comment:
It was introduced here:
https://github.com/kubernetes/kubernetes/pull/89438/files

This backoff is used when we fail the operation:
https://github.com/kubernetes/kubernetes/pull/89438/files#diff-8765679aa8ca8cc500a6dc038b8707cbc29ddc2d2f94f1da911e2cd5e4df5816R104

If we can't update ES within 100s backoff, we're really in big troubles or this is some kind of permanent error (like too big object or sth). Extending this to 1000s in those cases makes sense to me.

Although I'm not sure if the original numbers where backed by some real world scenarios...